fred2/musicplayerdlg.cpp

```diff
@@ -0,0 +1,214 @@
+// MusicPlayerDlg.cpp : implementation file
+//
+
+#include "stdafx.h"
+#include "FRED.h"
+#include "MainFrm.h"
+#include "musicplayerdlg.h"
+#include "TextViewDlg.h"
+#include "mod_table/mod_table.h"
+#include "sound/audiostr.h"
+
+#ifdef _DEBUG
+#undef THIS_FILE
+static char THIS_FILE[] = __FILE__;
+#endif
+
+music_player_dlg::music_player_dlg(CWnd* pParent /*=nullptr*/)
+	: CDialog(music_player_dlg::IDD, pParent)
+{
+
+	m_music_item = "";
+	m_music_id = -1;
+	m_cursor_pos = -1;
+	m_autoplay = FALSE;
```

Comment:
Default to autoplay off?  Do you believe this is what most users will want?

Reply:
It's what I would want. Most of the time I would use this to pick music for missions rather than just use it to listen to music while I FRED.